torch/_export/serde/serialize.py

```diff
@@ -315,15 +315,26 @@ def deserialize_torch_artifact(serialized: Union[Dict[str, Any], Tuple[Any, ...]
     return artifact
 
 
-def _sympy_int_to_int(val: sympy.Expr):
+def _sympy_int_to_int(val: sympy.Expr, adjust: str):
     # Convert simple sympy Integers into concrete int
     if val == sympy.oo:
         return math.inf
     if val == -sympy.oo:
         return -math.inf
     if isinstance(val, sympy.Integer):
         return int(val)
-    raise RuntimeError("Export constraints cannot be non-integer expressions")
+    log.warning(
```

Comment:
@angelayi can you leave a TODO here so that we can maybe remove this someday?

Reply:
done